world.rb

```diff
@@ -7,19 +7,31 @@ def initialize
   end
 
   def move_north(player)
-    player.y_pos -= 1 if player.y_pos > 0 else puts "You can't go that way"
```

Comment:
I prefer the one-liner, but I see that you were getting some warning. Can you change this and the three below to use a ternary operator instead of the `if ... else` one liner?

Like this: `player.y_pos > 0 ? player.y_pos -= 1 : puts "You can't go that way"`